committers.md

```diff
@@ -64,6 +64,7 @@ navigation:
 |Burak Yavuz|Databricks|
 |Matei Zaharia|Databricks, Stanford|
 |Shixiong Zhu|Databricks|
+|Anirudh Ramanathan|Google|
```

Comment:
not a big deal but I thought we keep this list in an order of the last name.

Reply:
Fixed, thanks!